www_src/pages/make/make.jsx

```diff
@@ -24,14 +24,9 @@ var Make = React.createClass({
   },
   render: function () {
     return (
-      <div id="make" className="demo">
-        <Link url="/map/123" href="/pages/map" className="tile tile-demo">
-          <div className="inner">
-            <div className="img">
-              <img src="../../img/toucan.svg" />
-            </div>
-          </div>
-        </Link>
+      <div id="make">
+        <button className="btn full teal">+ Create a Project</button>
+        <Tile/>
         <p><button onClick={this.onClick}>{this.state.test ? 'on' : 'off'}</button></p>
```

Comment:
Ah, this was a test thing for testing state persistence in android. feel free to remove